src/components/Main/VideoListItem/VideoListItem.jsx

```diff
@@ -44,15 +46,21 @@ const VideoListItem = (props) => {
           className={`up ${upLike ? 'active' : ''}`}
           onClick={() => props.likeVideo(upLike ? 0 : 1)}
           title={props.video.likes.filter(like => like.value === 1)
-            .map(like => like.user.username).join(', ')}
-        />
+            .map(like => like.user.username).join(', ')}>
+          <svg className="svg-icon">
```

Comment:
Maybe it would make sense to create a simple common component to wrap those three rows.